src/test/java/io/vertx/core/http/impl/HttpUtilsTest.java

```diff
@@ -229,4 +231,14 @@ private void check(String base, String ref, String expected) throws Exception {
     URI uri = HttpUtils.resolveURIReference(base, ref);
     assertEquals(expected, uri.getPath());
   }
+
+  @Test
+  public void testParams() {
```

Comment:
this should also be tested in `HttpTest`

Reply:
done